test/CancellationHandleTest.cpp

```diff
@@ -20,6 +22,10 @@ using ::testing::HasSubstr;
 using namespace std::chrono_literals;
 
 ad_utility::source_location location = ad_utility::source_location::current();
+const int expected_location_line = __LINE__ - 1;
+std::string expected_location() {
+  return "CancellationHandleTest.cpp:" + std::to_string(expected_location_line);
+}
```

Comment:
```suggestion
auto expectedLocation = absl::StrCat("CancellationHandleTest.cp:", expected_location_line);
```
(include `<absl/strins/str_cat.h>`

Reply:
Done